src/components/UploadEditor.vue

```diff
@@ -64,6 +64,8 @@
 				<NcButton type="tertiary" @click="handleDismiss">
 					{{ t('spreed', 'Dismiss') }}
 				</NcButton>
+				<!-- FIXME replace with NcRichContenteditable -->
+				<input v-model="mediaCaption">
```

Comment:
Should use NcContentEditable aka the NewMessageForm similar to 
https://github.com/nextcloud/spreed/blob/master/src/components/BreakoutRoomsEditor/SendMessageDialog.vue#L28